packages/soci-snapshotter/soci-snapshotter.spec

```diff
@@ -0,0 +1,54 @@
+%global gorepo soci-snapshotter
+%global gover 0.5.0
+%global rpmver %{gover}
+
+Name: %{_cross_os}soci-snapshotter
+Version: 0.5.0
+Release: 1%{?dist}
+Summary: Snapshots things
+License: Apache-2.0
+URL: https://github.com/awslabs/soci-snapshotter
+Source0: https://github.com/larvacea/soci-snapshotter/archive/refs/tags/v0.5.0.tar.gz
+Source1: bundled-v0.5.0.tar.gz
+Source2: bundled-cmd.tar.gz
+Source101: soci-snapshotter.service
+Source1000: clarify.toml
+
+BuildRequires: %{_cross_os}glibc-devel %{_cross_os}libz-devel
+
+Requires: %{_cross_os}containerd
+
+%description
+%{summary}.
+
+%prep
+%setup -n %{gorepo}-%{gover} -q
+%setup -T -D -n %{gorepo}-%{gover} -b 1 -q
+%setup -T -D -n %{gorepo}-%{gover} -b 2 -q
+
+%build
+
+%set_cross_go_flags
+
+(cd cmd ; go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o "../out/soci-snapshotter-grpc" ./soci-snapshotter-grpc)
+(cd cmd ; go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o "../out/soci" ./soci)
```

Comment:
We don't start subshells when we build multiple binaries. Instead of `cd`, you can tell `go` to compile from a path. See [this] for reference.

[this]: https://github.com/bottlerocket-os/bottlerocket/blob/507c21fa421f073861480be967ea8d053561e18b/packages/amazon-ssm-agent/amazon-ssm-agent.spec#L30